scripts/common.py

```diff
@@ -63,6 +67,8 @@ def __repr__(self):
 
 NAME_KEY = 'name'
 ISS_KEY = 'iss'
+WEBSITE_KEY = 'website'
+CANONICAL_ISS_KEY = 'canonicalIss'
```

Comment:
Since we seem to be using `participating_issuers`, should this also be snake-case `canonical_iss`?

Reply:
Good call, changed